src/Raven.Server/Rachis/CandidateAmbassador.cs

```diff
@@ -72,7 +72,7 @@ public void Dispose()
                     {
                         _engine.Log.Info(
                             $"CandidateAmbassador for {_tag}: Waited for a full second for thread {_candidateAmbassadorLongRunningWork.ManagedThreadId} " +
-                            $"({(_candidateAmbassadorLongRunningWork.Join(0) ? "running" : "finished")}) to finish, after the elections were {_candidate.ElectionResult}");
+                            $"({(_candidateAmbassadorLongRunningWork.Join(0) ? "finished" : "running")}) to finish, after the elections were {_candidate.ElectionResult}");
```

Comment:
finished to finish? running to finish?

Reply:
The issue was that we had an a polarity issue in the log statement.
If `Join` return `true`, we are finished, not running

Reply:
The message looks something like this:
`Waited for a full second for thread 123 (running) to finish`